browser/ui/views/frame/brave_browser_view.h

```diff
@@ -78,7 +84,7 @@ class BraveBrowserView : public BrowserView {
 #endif
 
   views::View* sidebar_host_view() { return sidebar_host_view_; }
-
+  void AddedToWidget() override;
```

Comment:
Please move this up to L87, so that it is in the overriden method section together.